.github/workflows/build-test.yml

```diff
@@ -69,7 +69,8 @@ jobs:
             lua5.3 liblua5.3-dev \
             libglfw3 libglfw3-dev \
             libglew-dev \
-            libglm-dev
+            libglm-dev \
+            libomp.dev
```

Comment:
https://packages.ubuntu.com/jammy/libomp-dev

Reply:
I tried that.